pkg/operator/ceph/file/filesystem.go

```diff
@@ -72,6 +73,11 @@ func createFilesystem(
 		}
 	}
 
+	err := cephclient.CreateCephFSSubVolumeGroup(context, clusterInfo, fs.Name, defaultCSISubvolumeGroup)
```

Comment:
Also make sure of it's deletion. https://github.com/red-hat-storage/ocs-operator/pull/2271/files#diff-fabbfe52f64e79da27f9fff14bbd71e9a6bc122fd84073a8a1a7debc4aa9ac30R168